src/99-appendices/03-hed.md

```diff
@@ -3,207 +3,175 @@
-               Item/Object/Vehicle/Aircraft/Airplane, Participant/Effect/Cognitive/Target,
-               Sensory presentation/Visual/Rendering type/Screen/2D),
-               (Item/Natural scene/Aerial/Satellite,
-               Sensory presentation/Visual/Rendering type/Screen/2D)",
-           "Button": "Event/Category/Participant response, Event/Label/PressButton,
-               Event/Description/The participant presses the button as soon as the target is visible,
-               Action/Button press"
-        }
-   }
-}
-```
-
-## Annotating events by value type
-
-Each column of `*_events.tsv` containing non-categorical values usually represents a
-particular type of data, for example the `speed` of a stimulus object across the
-screen or the filename of the stimulus image.
-These variables could be annotated in the HED column of `*_events.tsv`.
-However, that approach requires repeating the values appearing in the individual
-columns in the HED column.
-A better approach is to annotate the type of value contained in each of these
-columns in the `*_events.json` sidecar.
-Value variables are annotated in a manner similar to categorical values,
-except that the HED string must contain exactly one `#` specifying a placeholder
-for the actual column values.
-Tools are responsible for substituting the actual column values for the `#` during analysis.
-
-Example: An `*_events.tsv` containing a categorical column (`trial_type`) and two value
-columns (`response_time` and `stim_file`).
+Example: An excerpt from an `*_events.tsv` containing three columns
+(`trial_type`, `response_time`, and `stim_file`) in addition to
+the required `onset` and `duration` columns.
 
 ```Text
 onset  duration  trial_type  response_time stim_file
 1.2    0.6       go          1.435         images/red_square.jpg
 5.6    0.6       stop        1.739         images/blue_square.jpg
 ```
 
-Example: An accompanying `*_events.json` sidecar describing both categorical and value columns.
+The `trial_type` column in the above example contains a limited number of distinct
+values (`go` and `stop`).
+This type of column is referred to as a *categorical* column,
+and the column's meaning can be annotated by assigning HED tags to describe
+each of these distinct values.
+The JSON sidecar provides a dictionary of annotations for these categorical values.
+
+In contrast, the `response_time` and `stim_file` columns could potentially contain
+distinct values in every row.
+These columns are referred to as *value* columns and are annotated by creating
+a HED tag string to describe a general pattern for these values.
+The HED annotation for a value column must include a `#` placeholder,
+which tools will replace by the actual column value when the annotations
+are assembled for analysis.
+
+Example: An accompanying `_events.json` sidecar describing both categorical and
+value columns of the previous example.
+The `Duration` column is also annotated as a value column.
```

Comment:
I think I'd rather point out why the `onset` column is not annotated here.


Also: captialization --> 
```suggestion
The `duration` column is also annotated as a value column.
```